src/less/combobox/ds4/combobox.less

```diff
@@ -1,89 +1,58 @@
 @import "../../less/ds4/variables.less";
 @import "../../less/ds4/mixins.less";
 @import "../../dropdown/ds4/dropdown.less";
-@import "../base/combobox.less";
-
-@ds4-combobox-height: 40px;
 
-span.combobox {
-    vertical-align: top;
+.combobox-icon() {
+    .icon-arrow-down(9px, 100%);
 }
 
-.combobox__control > input {
-    background-color: @ds4-color-core-white;
-    border-color: @ds4-color-core-gray-silver;
-    border-radius: 3px;
-    font-size: @ds4-font-size-14;
-    height: @ds4-combobox-height;
-    padding: 0 32px 0 16px;
+.combobox-input-disabled-icon() {
+    .icon-arrow-down-disabled(9px, 100%);
+}
 
-    &[readonly] {
-        // fixes the cursor in firefox (2 of 2)
-        text-shadow: 0 0 0 @ds4-color-core-gray-davys;
+@combobox-height: 40px;
 
-        &::-moz-selection {
-            background: @ds4-color-core-white;
-        }
+@combobox-input-background-color: @ds4-color-core-white;
+@combobox-input-border-color: @ds4-color-core-gray-silver;
+@combobox-input-border-radius: 3px;
+@combobox-input-font-size: @ds4-font-size-14;
+@combobox-input-height: @combobox-height;
+@combobox-input-padding: 0 32px 0 16px;
 
-        &::selection {
-            background: @ds4-color-core-white;
-        }
-    }
+@combobox-input-readonly-text-shadow-color: @ds4-color-core-gray-davys;
+@combobox-input-readonly-moz-selection-background: @ds4-color-core-white;
```

Comment:
Why have a variable for @combobox-input-readonly-moz-selection-background with the same value as @combobox-input-readonly-selection-background

Can combine both and comine in combobox.less
 &::-moz-selection,
&::selection {
            background: @combobox-input-readonly-selection-background;
}

Reply:
👍 Will combine.